spec/requests/graphql_controller_spec.rb

```diff
@@ -33,17 +33,6 @@
     end
   end
 
-  context 'when variables are not a string or hash' do
```

Comment:
Do you remember why was this added in the first place?
I'm checking the commit but it doesn't ring any bells :sweat_smile:

Reply:
Yes, I added that just to increase coverage, but not sure how the Argument error was being raised in a request spec. Anyway, we need to fix the integration with coveralls anyway as I think it might not be setup properly with their latest changes. Also, the GQL gem probably has a different method for variable parsing now that might not raise an argument error. Thanks for the review